conans/client/tools/system_pm.py

```diff
@@ -74,6 +66,15 @@ def _create_tool(os_info, output):
         else:
             return NullTool(output=output)
 
+    def _get_sysrequire_mode(self):
```

Comment:
Good, but next time, do not move the method, because the diff is much harder to read.